cuda/private/actions/compile.bzl

```diff
@@ -9,7 +9,8 @@ def compile(
         srcs,
         common,
         pic = False,
-        rdc = False):
+        rdc = False,
+        _prefix = "_objs"):
```

Comment:
Aren't you missing the change in this function to actually make use of this prefix?

Reply:
Yes! Burried in another branch.